tests/JsonSchemaTest.php

```diff
@@ -206,4 +207,24 @@ public function testPayloadCollaborationWithInvalidJson()
         self::assertInstanceOf(ResponseInterface::class, $response);
         self::assertGreaterThanOrEqual(400, $response->getStatusCode(), $response->getReasonPhrase());
     }
+
+    public function testCustomErrorHandler()
+    {
+        $request = $this->request('/en/v1/users')
+            ->withParsedBody(json_decode(json_encode([
```

Comment:
i'm more comfortable writing schema as an array, but the underlying validator requires that the parsed body be a nested object. `json_decode(json_encode($arr))` seems to be the "easiest" way to recursively cast. it is certainly the laziest that i've seen : P